src/components/layouts/featurelist/feature-list-content.tsx

```diff
@@ -0,0 +1,450 @@
+            </>
+          ),
+          icon: CheckCircle
+          },
+          {
+            title: 'OpenID Certified®',
+            description: (
+            <>
+            Rely on an <Button style={'link-inline'} to={'https://openid.net/developers/certified/'}> OpenID Certified® OIDC Provider</Button>. Ory Hydra implements all flows specified by the IETF and OpenID Foundation.
+            </>
+          ),
+          icon: CheckCircle
+          },
+          {
+            title: 'Bring Your Own UX',
+            description: (
+            <>
+            Use your own branding and user interfaces for all OAuth2.0 and OIDC flows. Ory Hydra lets you use your styles and flows powered by a REST API and intuitive CLI.
+            </>
+          ),
+          icon: CheckCircle
+          },
+          {
+            title: 'Compatible with MITreid',
+            description: (
+            <>
+            Migrate from <Button style={'link-inline'} to={'https://mitreid-connect.github.io/'}>MITREid Connect</Button> to Ory Hydra. Migration documentation is provided.
+            </>
+          ),
+            icon: CheckCircle
+          },
+          {
+            title: 'Cryptographic Key Storage',
+            description: (
+            <>
+            Store cryptographic keys for e.g. signing JWTs securely and manage OAuth 2.0 clients directly from the CLI.
+            </>
+          ),
+            icon: CheckCircle
+          },
+          {
+            title: 'Security First and High Performance',
+            description: (
+            <>
+            Sleep easy, knowing that Ory Hydra is secured against attack vectors and scales as required. Ory Hydra serves tokens to millions of users weekly and just works.
+            </>
+          ),
+            icon: CheckCircle
+        }
+      ]
+    }
+  ]
+}
+
+
+export const featureListKeto  = {
+  features : [
+    {
+      icon: CircleThreePlus,
+      title: 'Ory Keto Features',
+      description: (
+        <>
+        Control who can access what in your application. Ory Keto serves decisions at scale globally with low latency.
+        </>
+      ),
+      button: (
+        <Button
+        style={'link'} to={'/docs/keto/quickstart/'} iconRight={ArrowRight} > Get started </Button>
+      ),
+      features: [
+        {
+          title: 'Manage and Check Permissions',
+          description: (
+            <>
+            Determine if a user is allowed to take an action with HTTP and gRPC APIs.
+            </>
+          ),
+          icon: CheckCircle
+          },
+          {
+            title: 'Low Latency and Zero Downtime',
+            description: (
+              <>
+              Delight users with a smooth experience and no downtime.
```

Comment:
What is a smooth experience? Hit them with facts :)

```suggestion
              Zanzibar has maintained 95th-percentile latency of less than 10 milliseconds and availability of greater than 99.999% over 3 years of production use.
```

Reply:
How does this sound? 

Ory Keto is based on [Zanzibar](https://www.ory.sh/keto-zanzibar-evolution/), which has maintained 95th-percentile latency of less than 10 milliseconds and availability of greater than 99.999% over 3 years of production use.

Reply:
Much better!